Проблема с подключением по IPv6

У меня на роутере настроен IPv6 и станция получает такой адрес. Binding не может подключиться к станции по IPv6 (telnet подключается без проблем). 
Нужно либо здесь исключать IPv6 адреса:
https://github.com/d51x/org.openhab.binding.yandexstation/blob/3bb751470c4f923903203bc8f0514754ef3b0b6a/src/main/java/org/openhab/binding/yandexstation/internal/YandexStationHandler.java#L648-L649
Либо искать проблему в клиенте. Подключение по IPv4 работает.

```
2023-11-15 02:33:49.004 [INFO ] [tation.internal.YandexStationHandler] - Yandex station IP's: [fd76:2458:4d33:0:fcf2:458b:3810:3b57, 192.168.1.19]
2023-11-15 02:33:49.009 [INFO ] [tation.internal.YandexStationHandler] - Connect to Yandex Station: *** with IP fd76:2458:4d33:0:fcf2:458b:3810:3b57
2023-11-15 02:33:49.010 [WARN ] [tation.internal.YandexStationHandler] - Try connect after: 60 sec
2023-11-15 02:34:49.014 [WARN ] [ty.util.ssl.SslContextFactory.config] - Trusting all certificates configured for Client@4d8915de[provider=null,keyStore=null,trustStore=null]
2023-11-15 02:34:49.014 [WARN ] [ty.util.ssl.SslContextFactory.config] - No Client EndPointIdentificationAlgorithm configured for Client@4d8915de[provider=null,keyStore=null,trustStore=null]
2023-11-15 02:34:49.015 [ERROR] [tation.internal.YandexStationHandler] - Connection error null
2023-11-15 02:34:49.016 [WARN ] [tation.internal.YandexStationHandler] - Try connect after: 60 sec
```

--- a/src/main/java/org/openhab/binding/yandexstation/internal/YandexStationHandler.java
+++ b/src/main/java/org/openhab/binding/yandexstation/internal/YandexStationHandler.java
@@ -646,7 +646,7 @@ public class YandexStationHandler extends BaseThingHandler {
 
             // config.hostname = device.networkInfo.ipAdresses.get(0);
             config.hostname = Objects.requireNonNull(device.networkInfo.ipAdresses.stream()
-                    .filter(ip -> !ip.startsWith("169.254")).findFirst().orElse(null));
+                    .filter(ip -> !ip.startsWith("169.254")).filter(ip -> !ip.contains(":")).findFirst().orElse(null));
             config.port = String.valueOf(device.networkInfo.port);
 
             Configuration configuration = thing.getConfiguration();
@@ -677,7 +677,7 @@ public class YandexStationHandler extends BaseThingHandler {
         properties.put("Wifi SSID:", device.networkInfo.wifiSSID);
         // properties.put("IP Address:", device.networkInfo.ipAdresses.get(0));
         properties.put("IP Address:", Objects.requireNonNull(device.networkInfo.ipAdresses.stream()
-                .filter(ip -> !ip.startsWith("169.254")).findFirst().orElse(null)));
+                .filter(ip -> !ip.startsWith("169.254")).filter(ip -> !ip.contains(":")).findFirst().orElse(null)));
         properties.put("Platform:", device.platform);
         properties.put("Device Name:", YandexStationTypes.getNameByPlatform(device.platform));
         properties.put("Friendly Name:", device.name);

